edge-runtime/lib/response.ts

```diff
@@ -197,11 +195,10 @@ export const buildResponse = async ({
     return addMiddlewareHeaders(context.rewrite(target), res)
   }
 
-  // If we are redirecting a request that had a locale in the URL, we need to add it back in
-  if (redirect && requestLocale) {
-    redirect = normalizeLocalizedTarget({ target: redirect, request, nextConfig, requestLocale })
+  if (redirect) {
+    redirect = normalizeLocalizedTarget({ target: redirect, request, nextConfig })
     if (redirect === request.url) {
-      logger.withFields({ rewrite_url: rewrite }).debug('Rewrite url is same as original url')
+      logger.withFields({ redirect_url: redirect }).debug('Redirect url is same as original url')
```

Comment:
this seems like was previously c&p-ed from rewrite handling so log fields/msg was off